frontend/src/app/modules/login/forget-password/forget-password.component.sass

```diff
@@ -0,0 +1,68 @@
+@import 'color-variables'
+
+.page-wraper
+    display: flex
+    flex-direction: column
+    align-items: center
+    color: $white
+    gap: 16px
+
+.header
+    font-size: 24px
+    font-weight: 300
+    font-family: "Open Sans"

```

Comment:
could you please add this to in global HTML to the body so we do not need to add this to other elements?

Reply:
Added